### PR TITLE
Fused dropout in multi head attention

### PR DESCRIPTION
**Pull request by cmdevries**
Fused dropout in multi head attention is twice as fast as the torch implementation. However, this barely makes any difference to the overall efficiency of the forward pass as it reduces GPU time from 26 microseconds to 13 microseconds for the operation. Training loss behaves similarly to before the change.

### CODE REVIEW

File: xformers/components/multi_head_dispatch.py [hunk 1]
```diff
@@ -13,6 +13,7 @@
 from torch.nn.init import constant_
 
 from xformers.components.attention import Attention
+from xformers.triton import FusedDropoutBias
```
Comment (vmarkovtsev) [inline xformers/components/multi_head_dispatch.py]: Would you pls sort imports :pray:

**Review by vmarkovtsev — CHANGES_REQUESTED** [review]

**Review by vmarkovtsev — APPROVED** [review]

File: xformers/components/multi_head_dispatch.py (hunk 1 above)
Comment (cmdevries) [reply xformers/components/multi_head_dispatch.py]: Done
